.github/workflows/validate_packs.yml

```diff
@@ -49,3 +49,20 @@ jobs:
       - name: Check for non-conventional commits
         run: ./utils/conventional-commits.sh
 
+  ensure-images-are-valid:
```

Comment:
i think there is a default you can set for working directory.

something like:
```yml
jobs:
  x-job:
    defaults:
      working-directory: utils
```

im not 100% thats correct, but it should be something like that.

Reply:
just a heads up, no need to change this. just giving some insight.